src/Reactduino.cpp

```diff
@@ -164,6 +195,26 @@ reaction Reactduino::onInterrupt(uint8_t number, react_callback cb, int mode)
     return r;
 }
 
+reaction Reactduino::onInputChange(uint8_t pin, react_callback cb, int state)
+{
+    reaction r;
+
+    r = alloc(REACTION_TYPE_INPUT_CHANGE, cb);
+
+    if (r == INVALID_REACTION) {
+        return INVALID_REACTION;
+    }
+
+    input_change_specs_t specs;
```

Comment:
Could we move this line up to 201 (after `reaction r;`).

It's a bit of an old C habbit but I usually stick to the convention of declaring variables at the beginning of a code block.